js_modules/dagster-ui/packages/ui-components/src/components/NewConfigEditor.tsx

```diff
@@ -194,7 +194,7 @@ export const NewConfigEditor = forwardRef<ConfigEditorHandle, ConfigEditorProps>
       <StyledRawCodeMirror
         value={configCode}
         theme={['config-editor']}
-        options={options}
+        options={options as any}
```

Comment:
Is this avoidable?

Reply:
I'll remove it, don't need it now. But no it's not because of the patched lint thing. It's expecting options that are not part of the official lint addon and so the types that the addon has cause an error.